contrib/http_base/README.md

```diff
@@ -0,0 +1,4 @@
+OpenTelemetry Contrib Http Support
+======================================================
+
+* Java 7 compatible.
```

Comment:
Just curious..do we need all contrib to be java 7 compatible?

Reply:
Good question. I left as Java 7 for this module because I could see it being used in Android but maybe that is the wrong choice.

Reply:
@bogdandrutu  provided an answer in https://github.com/open-telemetry/opentelemetry-java/pull/404#issuecomment-502123153